src/map/packets/char.cpp

```diff
@@ -32,7 +32,7 @@
 CCharPacket::CCharPacket(CCharEntity* PChar, ENTITYUPDATE type, uint8 updatemask)
 {
     this->type = 0x0D;
-    this->size = 0x3A;
+    this->size = 0x5C;
```

Comment:
I was honestly just cargo-culting and copying the sizes I observed in captures. It's probably worth trying to cut down all packets to be as small as we can get away with

Reply:
I suggest an overload that does the version you need - some retail packets have multiple alternates in such a manner 
https://github.com/LandSandBoat/server/blob/base/src/map/packets/treasure_lot_item.cpp#L40

though my example is not a great one because the one I linked is now outdated (retail changed what it does, but the old versions still work - lucky for us)